### DISCUSSION TITLE
A bug in RethinkDNS search?

### DISCUSSION THREAD

**Discussion by MasterKia** [serverless-dns/serverless-dns#129] [Q&A]
There ~~is~~ was an incorrect entry in my list ([PersianBlockerHosts](https://github.com/MasterKia/PersianBlocker/blob/3c3062c973b43258620a5f6ef78d4362117f6aca/PersianBlockerHosts.txt#L520)) which I've removed:
`t.me/BITBOD_TRADING_CLUB`

When I search for `t.me` in `rethinkdns.com/search`, it finds a result in my list too but logically RethinkDNS should discard any invalid entry in filterlists (as is `t.me/BITBOD_TRADING_CLUB`); but instead it counts them as valid hostnames?

https://rethinkdns.com/search?q=t.me

@ignoramous

**Comment by ignoramous** [comment]
A domain list or a host list shouldn't have URLs. 

Blocklist maintainers usually maintain a domains-only / hosts-only versions.

**Reply by MasterKia** [reply]
My understanding is that RethinkDNS somehow converted that URL into a hostname and enforced it?

**Reply by ignoramous** [reply]
Yes, it may have done that, but a domain / hosts file usually shouldn't contain URLs or Regexes.

**Comment by SpencerIsGiddy** [comment]
From my knowledge, the search gets updated every 1st, 8th, 16th, 24th of each month which is when rethink updates the lists.

Was it still appearing after a “last updated” after you updated your list? For example, you updated your list feb 3rd and then the search says last updated feb 1st.  and you rechecked on the feb 10th which then it says last updated feb 8th and it still appeared when searching. (If that makes sense)
![0D863D16-C0EC-4D22-ABD4-745C3B64AD21](https://user-images.githubusercontent.com/109806759/219967619-8dac8b3b-0f0f-4524-ae00-e7bb399b48f2.jpeg)
